go.mod

```diff
@@ -70,8 +71,6 @@ replace (
 	github.com/Azure/go-autorest => github.com/Azure/go-autorest v14.0.0+incompatible
 	// github.com/aws/aws-sdk-go => github.com/weaveworks/aws-sdk-go v1.25.14-0.20191218135223-757eeed07291
 	github.com/awslabs/goformation => github.com/errordeveloper/goformation v0.0.0-20190507151947-a31eae35e596
-	// Override version since auto-detected one fails with GOPROXY
-	github.com/census-instrumentation/opencensus-proto => github.com/census-instrumentation/opencensus-proto v0.2.0
```

Comment:
This is not needed anymore?

Reply:
as far as I can tell, no. if I remove it from `master`, I get invalid version errors building (because of `goreleaser`), which I no longer get with this branch. And I don't have any problems while using GOPROXY either (I think it's used for the docker builds?).